nav2_amcl/src/amcl_node.cpp

```diff
@@ -568,51 +568,22 @@ AmclNode::setMapCallback(
 void
 AmclNode::laserReceived(sensor_msgs::msg::LaserScan::ConstSharedPtr laser_scan)
 {
-  std::string laser_scan_frame_id = strutils::stripLeadingSlash(laser_scan->header.frame_id);
   last_laser_received_ts_ = now();
   if (map_ == NULL) {
     return;
   }
 
+  std::string laser_scan_frame_id = strutils::stripLeadingSlash(laser_scan->header.frame_id);
```

Comment:
The removing leading slashes was a fix from TF->TF2 conversion. At this point, I think we should drop it and assume its well formed. TF1 isnt being ported to ROS2